src/api/auth.md

```diff
@@ -159,6 +159,8 @@ if there is a flag associated to it then at least one of them must be met.
 | {% break : payment-requests:read-by-shortcode   %} |       ✅       |               |         ✅         |                         |
 | {% break : payment-requests:refund 🗄 %}            |       ✅       |               |         ✅         |                         |
 | {% break : payment-requests:void 🗄 %}              |       ✅       |               |         ✅         |                         |
+| {% break : requests:release 🗄 %}                   |       ✅       |               |         ✅         |                         |
+| {% break : requests:confirm 🗄 %}                   |       ✅       |               |         ✅         |                         |
```

Comment:
Let's be consistent here and set the resource as `payment-requests`.

Reply:
+1, doesn't have to match internal lingo but should be externally consistent.